src/Components/Comments.jsx

```diff
@@ -15,6 +17,24 @@ export default function Comments({ article_id }) {
     });
   }, [comments]);
 
+  const handleDeleteComment = (event, comment_id) => {
+    event.preventDefault();
+    alert("You have deleted a comment!");
```

Comment:
I don't think this is the best way to provide feedback. Could we render something on screen perhaps?